src/test/appForm.test.js

```diff
@@ -309,6 +321,55 @@ describe("App Form", function () {
           FormActions.update("instances", 2);
         });
       });
+
+      describe("the executor field", function () {
+        it("updates correctly", function (done) {
+          AppFormStore.once(FormEvents.CHANGE, function () {
+            expectAsync(function () {
+              expect(AppFormStore.fields.executor).to.equal("//cmd");
+            }, done);
+          });
+
+          FormActions.update("executor", "//cmd");
+        });
+      });
+
+      describe("the ports field", function () {
+        it("updates correctly", function (done) {
+          AppFormStore.once(FormEvents.CHANGE, function () {
+            expectAsync(function () {
+              expect(AppFormStore.fields.ports).to.equal("23, 24, 25");
+            }, done);
+          });
+
+          FormActions.update("ports", "23, 24, 25");
```

Comment:
Would be nice to see the `app.ports` tested too. I suggest we refactor all these tests tomorrow and merge the `the model` and `the form fields` describe blocks so we can check both for each field.